pkg/renew/renew.go

```diff
@@ -96,24 +96,30 @@ func NewCmdRenew(ctx context.Context, ioStreams genericclioptions.IOStreams) *co
 
 // Validate validates the provided options
 func (o *Options) Validate(cmd *cobra.Command, args []string) error {
-	if len(o.LabelSelector) > 0 && len(args) > 0 {
-		return errors.New("cannot specify Certificate names in conjunction with label selectors")
+	if o.AllNamespaces && cmd.PersistentFlags().Changed("namespace") {
+		return errors.New("cannot specify --namespace flag in conjunction with --all-namespaces flag")
 	}
 
 	if len(o.LabelSelector) > 0 && o.All {
 		return errors.New("cannot specify label selectors in conjunction with --all flag")
 	}
 
-	if o.All && len(args) > 0 {
-		return errors.New("cannot specify Certificate names in conjunction with --all flag")
-	}
+	if len(args) == 0 {
+		if !o.All && len(o.LabelSelector) == 0 {
+			return errors.New("please either supply one or more Certificate resource names, label selectors, or use the --all flag to renew all Certificate resources")
+		}
+	} else {
+		if len(o.LabelSelector) > 0 {
+			return errors.New("cannot specify Certificate names in conjunction with label selectors")
+		}
 
-	if o.All && cmd.PersistentFlags().Changed("namespace") {
-		return errors.New("cannot specify --namespace flag in conjunction with --all flag")
-	}
```

Comment:
Providing --namespace and --all should not error, as described in the --all [description](https://github.com/cert-manager/cmctl/blob/c42c489dd0af74bb6a6568ea55e86cced39b4dc2/pkg/renew/renew.go#L90); it also matches the behaviour of the code